caluma/caluma_form/tests/__snapshots__/test_document.ambr

```diff
@@ -2,28 +2,8 @@
   <class 'dict'> {
     'allDocuments': <class 'dict'> {
       'edges': <class 'list'> [
-        <class 'dict'> {
-          'node': <class 'dict'> {
-            'answers': <class 'dict'> {
-              'edges': <class 'list'> [
-                <class 'dict'> {
-                  'node': <class 'dict'> {
-                    '__typename': 'DateAnswer',
-                    'date_value': '2019-02-22',
-                    'question': <class 'dict'> {
-                      'label': 'Taylor Williams',
-                      'slug': 'suggest-adult-allow',
-                    },
-                  },
-                },
-              ],
-              'totalCount': 1,
-            },
-            'createdByUser': 'a18c653a-d278-47f1-8d6e-9ced73aeeea4',
-          },
-        },
```

Comment:
why is this gone?

Reply:
https://github.com/projectcaluma/caluma/pull/1438/files#r773348231

filter uses None in a date-search